addons/fleet/models/fleet_make.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from openerp import api, fields, models, tools
+
+
+class FleetMake(models.Model):
+    _name = 'fleet.make'
+    _description = 'Make of the vehicle'
+    _order = 'name asc, id asc'
```

Comment:
no need to define explicit ascending order, _order = 'name, id' is enough(apply this at all the places in this branch)